docusaurus.config.js

```diff
@@ -0,0 +1,162 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require("prism-react-renderer/themes/github");
+const darkCodeTheme = require("prism-react-renderer/themes/dracula");
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: "werewolv.es",
+  favicon: "favicon.ico",
+
+  // Set the production url of your site here
+  url: "https://werewolv.es",
+  // Set the /<baseUrl>/ pathname under which your site is served
+  // For GitHub pages deployment, it is often '/<projectName>/'
+  baseUrl: "/guides",
```

Comment:
Matches what the old Docz/Gatsby setup did, but the deploy to surge didn't like it since it was at the root.